dali/driver/base.py

```diff
@@ -238,7 +241,7 @@ def close(self):
         super(USBListener, self).close()
 
 class SerialBackend(Backend):
-    """``Backend`` implementation for Quad-DALI-USB-Interface. Uses serial connection."""
+    """``Backend`` implementation to communicate with DALI interfaces over a serial connection."""
 
     def __init__(self, port, baudrate=115200, bytesize=serial.EIGHTBITS, parity=serial.PARITY_NONE,
```

Comment:
There's also an issue here, as I am using constants from ``serial`` in the constructor. This breaks with the import being in the constructor. Can't think of an easy fix, though.

Reply:
I am talking about line 243/246, hope that's clear. Haven't used this commenting feature before, so please bear with me.

Reply:
You can set the defaults to None and use the serial constants at init time:

    def __init__(self, bytesize=None):
        try:
            import serial
            bytesize = bytesize if bytesize is not None else serial.EIGHTBITS
        except ImportError:
            raise

Reply:
Thank you for that super elegant solution 👍